plasmapy/diagnostics/thomson.py

```diff
@@ -111,12 +129,39 @@ def spectral_density(
 
     electron_vel : `~astropy.units.Quantity`, shape (Ne, 3), optional
         Velocity of each electron component in the rest frame. (convertible to m/s)
+        If set, overrides electron_vdir and electron_speed.
         Defaults to a stationary plasma [0, 0, 0] m/s.
 
+    electron_vdir : np.ndarray, shape (Ne,3), optional
```

Comment:
```suggestion
    electron_vdir : `numpy.ndarray`, shape (Ne,3), optional
```

This will allow `sphinx` to properly link to the `numpy` documentation.